beets/ui/commands.py

```diff
@@ -39,6 +42,7 @@
 from beets.util.confit import _package_path
 
 VARIOUS_ARTISTS = u'Various Artists'
+ExtraChoice = namedtuple('ExtraChoice', ['plugin', 'id', 'long', 'callback'])
```

Comment:
This is basically for readability. The `plugin` field is not currently used - I left it there just in case the actual plugin is needed for something (for example, getting its name for displaying the user which choices are standard, and which choices are appended by that plugin).